src/components/pages/pricing/forecast/forecast.jsx

```diff
@@ -0,0 +1,159 @@
+'use client';
+
+import useScrollPosition from '@react-hook/window-scroll';
+import { Alignment, Fit, Layout, useRive, useStateMachineInput } from '@rive-app/react-canvas';
+import { useEffect, useRef, useState } from 'react';
+import { useInView } from 'react-intersection-observer';
+
+import Container from 'components/shared/container/container';
+import Link from 'components/shared/link';
+import LINKS from 'constants/links';
+import { activities, performance, storage } from 'constants/pricing';
+import useWindowSize from 'hooks/use-window-size';
+import ArrowIcon from 'icons/arrow-sm.inline.svg';
+
+import Metrics from './metrics';
+
+const SECTION_MIN_HEIGHT = 760;
+
+const getSelectedIndex = (activeTitle, items) =>
+  items.findIndex(({ title }) => title === activeTitle) + 1;
+
+const Forecast = () => {
+  const sectionRef = useRef();
+  const { width: windowWidth, height: pageHeight } = useWindowSize();
```

Comment:
Let's remove `windowWidth` as props of Metrics component
```suggestion
  const { height: pageHeight } = useWindowSize();
```